File: CE303/MarketCS/MarketServerCS/Trader.cs
namespace MarketServerCS
{
    class Trader {
    public Trader(int traderID, int clientID) {
        TraderID = traderID;
        ClientID = clientID;
    }
    
    public int TraderID {get;}
    public int ClientID {get;}
    public bool Stock {get; set;}
    
    }
}

File: CE303/src/MarketServerJava/Trader.java
package MarketServerJava;

public class Trader {
    private int traderID;
    private int clientID;
    private boolean stock;
    public Trader(int traderID, int clientID){ this.traderID=traderID; this.clientID = clientID; }

    public int getTraderID(){ return traderID; }

    public int getClientID(){ return clientID; }

    public boolean getStock(){ return stock; }

    public void setStock(boolean stock){ this.stock=stock; }


}
